src/index.tsx

```diff
@@ -26,6 +26,7 @@ type ImageSettings = {
   x?: number;
   y?: number;
   opacity?: number;
+  crossOrigin?: 'anonymous' | 'use-credentials' | '';
```

Comment:
This is absolutely correct but using union types can make using the library harder (it's why `level` is a string). I'll need to see what we can do about that.

Reply:
Landed #291 which did that and it's fine, so carry on.